tests/instrumentation-framework/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>hawkular-apm</artifactId>
+    <groupId>org.hawkular.apm</groupId>
+    <version>0.9.3.Final-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>instrumentation-test-framework</artifactId>
+
+  <properties>
+    <!-- override some version for spotify docker client -->
+    <version.com.fasterxml.jackson.core>2.6.0</version.com.fasterxml.jackson.core>
+    <version.org.apache.httpcomponents>4.5</version.org.apache.httpcomponents>
```

Comment:
Yes, but in this case I just wanted to override versions from the parent pom. Moving to the parent will cause update the versions for whole APM. Is that ok?